### PR TITLE
feat: Add new FR, docs, and Bug Report issues templates

### PR DESCRIPTION
**Pull request by leeederek**
In preparation for making our repository public, I am adding 3 new issues templates for our users to leverage when giving feedback to us. These new issues templates use Github's new issues form format.

### Rationale: 
Feedback from our users is very valuable because it helps our team learn faster and improve Kurtosis over time. Adding guard rails (via a more structured, easy-to-fill-out form) to the way our users give feedback to us on Github will reduce the iteration cycles needed to respond to our community, raise the quality bar for our responses, and ultimately reduce the time it takes to turn around bug fixes and new features for our users.

I have also removed the old generic template because:
- It contained links to internal pages that will not be accessible to our end users
- It did not distinguish between bug and FR and instead was very free-form
- It uses the older .md issues template format that is not as clean and seamless as the new Issues Form feature on Github

### CODE REVIEW

**Comment by leeederek** [comment]
Adding @mieubrisse and @victorcolombo for thoughts and +1s.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: "Bug Report"
+description: We are always striving to improve Kurtosis. Use this form to report a bug with Kurtosis.
+title: "[bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report. We value your feedback.
+        Please make sure to update the title of this Bug Report t to concisely describe the issue.
```
Comment (leoporoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: there is a typo here ...Report `t` to...

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - type: input
+    id: version
+    attributes:
+      label: What version of Kurtosis were you running when you encountered this bug?
+      placeholder: e.g. 0.67.0
+      description: |
+        To view the current version of Kurtosis you are using, run `kurtosis version`
```
Comment (leoporoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: nice!

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: priority
+    attributes:
+      label: What is the severity of this bug?
+      multiple: false
+      options:
+      - Critical, I am blocked and Kurtosis is unusable to us because of this bug.
```
Comment (leoporoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml:36]: nice!!

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report. We value your feedback.
+        Please make sure to update the title of this Bug Report t to concisely describe the issue.
+  - type: input
+    id: version
+    attributes:
+      label: What version of Kurtosis were you running when you encountered this bug?
```
Comment (adschwartz) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: Perhaps we should specify that this is for the CLI? If we have a cloud offering in the future there will be a whole different product there to file bugs for. We should probably take this time to specify which product we are asking for input on?

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (leoporoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: I'm seeing the `bug report` in the first sentence is lower case and upper case in the second sentence. Is this on purpose?

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
+name: "Feature Request"
+description: We are always striving to improve Kurtosis. Use this form to suggest a new feature or improvement. 
+labels: ["feature request"]
+title: "[FR]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this feature request form. We value your feedback.
```
Comment (leoporoli) [inline .github/ISSUE_TEMPLATE/feature-request.yml]: I'm seeing the `feature request` in the first sentence is lower case and upper case in the second sentence. Is this on purpose?

Comment (leeederek) [reply .github/ISSUE_TEMPLATE/feature-request.yml]: good catch, i will fix

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (leeederek) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: good catch, i will fix

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 4 above)
Comment (leeederek) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: That's a good point. Long term, I suspect we'll want a queue for open source questions/support and a separate, customer support channel/process for questions/support for our paying customers.

**Comment by leeederek** [comment]
Preview links!
- [Bug report](https://github.com/kurtosis-tech/kurtosis/blob/derek/add-new-issues-templates/.github/ISSUE_TEMPLATE/bug-report.yml)
- [Feature requests](https://github.com/kurtosis-tech/kurtosis/blob/derek/add-new-issues-templates/.github/ISSUE_TEMPLATE/feature-request.yml)

**Review by mieubrisse — APPROVED** [review]

**Pull request merged**